components/scripts/lib/build_scan.sh

```diff
@@ -154,7 +154,7 @@ fi
 fetch_and_read_build_scan_data() {
   # This isn't the most robust way to read a CSV,
   # but we control the CSV so we don't have to worry about various CSV edge cases
-  local args build_cache_metrics_only fetched_data header_row_read
+  local args build_cache_metrics_only fetched_data header_row_read idx
```

Comment:
What does it mean to pass `idx` here?

Reply:
This declares that `idx` is a local variable to this function. So after this function is done executing, it won't be accessible anymore.

Unlike other languages, which assume variables are local unless otherwise specified, bash assumes all variables are global (unless otherwise specified).

Reply:
Got it. Thanks for explaining, @jthurne.